packages/material-ui-lab/src/useAutocomplete/useAutocomplete.spec.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
nit: move these into a functions just so that we're safe from misinterpretation of usage.

Reply:
ok